packages/containers/Import/Import.js

```diff
@@ -190,6 +193,17 @@ class Import extends Component {
           currentStep={currentStep}
           onClickStep={this.handleClickStep}
         />
+        {currentStep === IMPORTING_STEP && (
+          <ButtonWrapper>
+            <Button
+              color="white"
+              disabled={importProgress < IMPORT_PROGRESS_THRESHOLD}
+              onClick={this.handleCancelFlow}
+            >
+              IMPORT ONE MORE *.CSV
```

Comment:
Let's write text in lowerCase with the first letter - upper.
Button component has default property `text-transform: uppercase;`.
So `Import one more *.csv` will be better.

Reply:
Done